man/systemd.network.xml

```diff
@@ -1969,6 +1969,14 @@ allow my_server_t localnet_peer_t:peer recv;</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>UseCaptivePortal=</varname></term>
+        <listitem>
+          <para>When true (the default), the captive portal advertised by the DHCP server will be recorded
+          and made available to client programs.</para>
```

Comment:
Maybe, better to explicitly mention `networkctl`.

Reply:
Ok, done.